src/common/transformations/src/transformations/common_optimizations/shared_ops_optimization.cpp

```diff
@@ -208,6 +260,15 @@ bool pass::SharedOpOptimization::run_on_model(const shared_ptr<Model>& model) {
         RECORD(v1::Reshape, reshapes_are_equal),
         RECORD(v0::ShapeOf, shapeof_are_equal),
         RECORD(v3::ShapeOf, shapeof_are_equal),
+        RECORD(v1::Divide, divide_are_equal),
+        RECORD(v1::Multiply, eltwise_arithmethic_are_equal),
```

Comment:
I would recommend to run a performance validation

Reply:
If there will be an activity to measure performance -- please include these changes with Transpose also:
https://github.com/openvinotoolkit/openvino/pull/18775/